Kitodo-API/src/main/java/org/kitodo/api/externaldatamanagement/ExternalDataImportInterface.java

```diff
@@ -52,6 +53,21 @@
      */
     SearchResult search(String catalogId, String field, String term, int rows);
```

Comment:
Isn't this a map with one entry? What's the benefit of having a separate method?